ui/apps/platform/src/Components/PatternFly/SearchFilterChips.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Flex, FlexItem, ChipGroup, Chip, Button, ToolbarChip } from '@patternfly/react-core';
```

Comment:
Small request to order named imports. Sometimes auto additions by editor are too invisible.

Lesser importance, but you might also order by hooks, types, utils below.